lib/ghost_adapter.rb

```diff
@@ -5,7 +5,7 @@
 
 require 'ghost_adapter/config'
 
-require 'ghost_adapter/railtie' if defined? ::Rails::Railtie
+require 'ghost_adapter/railtie' if defined? Rails::Railtie
```

Comment:
Rubocop had issues with the `::` prefix. @acroos did you put them there for a reason? If so I'll update with a Rubocop ignore tag.

Reply:
If there was a reason, I certainly can't recall it.  If this still works then I'm going to remove it!